YACReader/main_window_viewer.cpp

```diff
@@ -1596,6 +1476,46 @@ void MainWindowViewer::getSiblingComics(QString path, QString currentComic)
     updatePrevNextActions(index > 0, index + 1 < list.count());
 }
 
+void MainWindowViewer::setActionsEnabled(bool enabled)
+{
+    //TODO enable goTo and showInfo (or update) when numPages emited
```

Comment:
This pull request enables only the actions that used to be handled by `keyPressEvent()`s when `Render::numPages` => `Viewer::comicLoaded` signal is emitted. This old [merely moved in this PR] comment suggests that a couple more actions should be enabled at this signal rather than when they are now - when opening/loading/extracting a comic only begins. I wonder, perhaps *all* these actions should be enabled at the later time when the comic is completely loaded/extracted?

If not, then at least one call to `enableActions()` should be removed as redundant from `MainWindowViewer::open(QString path, qint64 comicId, qint64 libraryId)`, because it either fails or calls `MainWindowViewer::open(QString path, ComicDB &comic, QList<ComicDB> &siblings)`, which in turn also calls `enableActions()`.

Reply:
@luisangelsm, what do you think about the issue described in this comment? This PR preserved the existing times when the actions were enabled/disabled. But since the implementation had been nonuniform, the difference might be unintended. The said difference is now reflected in the different places where `enableAcitons()` and `setLoadedComicActionsEnabled()` are invoked. I am wondering if all these actions should be handled uniformly. Or perhaps at least some actions should be moved between the two groups.